src/plugins/vis_augmenter/server/saved_objects/augment_vis.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectsType } from 'opensearch-dashboards/server';
+
+export const augmentVisSavedObjectType: SavedObjectsType = {
+  name: 'augment-vis',
+  hidden: false,
+  namespaceType: 'single',
+  mappings: {
+    properties: {
+      description: { type: 'text' },
+      pluginResourceId: { type: 'text' },
+      visName: { type: 'keyword', index: false, doc_values: false },
```

Comment:
`index: false` means the field is stored but not indexed or available for search. `doc_values: false` means the field does not support aggregation or sorting, or accessible via a script, which saves disk space. Because `visName` is an internal value set via `injectReferences()`/`extractReferences()`, it is only needed to be stored. This follows the pattern of `savedSearchRefName` in the `visualization` saved object: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/visualizations/server/saved_objects/visualization.ts#L63